index.js

```diff
@@ -25,6 +25,9 @@ function mewt(target) {
       return mewt(newObj)
     },
     $unset: prop => {
+      if (isA && Number.isInteger(prop)) {
+        return mewt([...target.slice(0, prop), ...target.slice(prop + 1)])
```

Comment:
Sure, but what do you say if I make the test temporarily ignored, and we do it in a separate PR? So this can be merged and changes don't pile up?